src/main/java/com/spring/demo/configs/LoggingServiceActivator.java

```diff
@@ -0,0 +1,24 @@
+package com.spring.demo.configs;
+
+import org.springframework.integration.annotation.ServiceActivator;
+import org.springframework.integration.support.MessageBuilder;
+import org.springframework.messaging.Message;
+import org.springframework.messaging.MessagingException;
+import org.springframework.stereotype.Component;
+
+@Component
+public class LoggingServiceActivator {
+
+  @ServiceActivator(inputChannel = Names.LOG_CHANNEL)
+  public Message<Object> logger(Message<Object> request) throws MessagingException {
+
+    System.out.println(request.toString());
```

Comment:
using Logger instead of System.out.println
  example:
  traditional style:
`  
private static final Logger log = LoggerFactory.getLogger(LoggingServiceActivator.class);
`
or Lombok style: 
`@Slf4j`
then 
`
log.info("Input request: {}", request.toString());
`